Create icon-tint-effect-behavior.md (#115)
MicrosoftDocs/CommunityToolkit: docs/maui/behaviors/icon-tint-color-behavior.md

---
title: IconTintColorBehavior - .NET MAUI Community Toolkit
author: vhugogarcia

> description: "The IconTintColorBehavior allows you to tint an image with an attached property."

Reviewer: ```suggestion
description: "The IconTintColorBehavior is a behavior allows you to tint an image."
```

ms.date: 06/27/2022
---

> 

Reviewer: Do not merge until we merge and release: 

https://github.com/CommunityToolkit/Maui/pull/456
